integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -307,6 +324,277 @@ public void testUpdateDataSourceInDomainUsingAuxiliaryImage() {
+  public void testErrorPathDomainMismatchMountPath() {
+
+    OffsetDateTime timestamp = now();
+    String errorPathAuxiliaryImage1 = MII_AUXILIARY_IMAGE_NAME + ":errorpathimage1";
+
+    final String auxiliaryImageVolumeName = "auxiliaryImageVolume1";
+    final String auxiliaryImagePath = "/errorpath";
+
+    createSecretsForDomain(adminSecretName, encryptionSecretName, errorpathDomainNamespace);
+
+    // create stage dir for auxiliary image
+    Path errorpathAIPath1 = Paths.get(RESULTS_ROOT, "errorpathauxiimage1");
+    assertDoesNotThrow(() -> FileUtils.deleteDirectory(errorpathAIPath1.toFile()));
+    assertDoesNotThrow(() -> Files.createDirectories(errorpathAIPath1));
+
+    // create models dir and copy model for image
+    Path modelsPath1 = Paths.get(errorpathAIPath1.toString(), "models");
+    assertDoesNotThrow(() -> Files.createDirectories(modelsPath1));
+    assertDoesNotThrow(() -> Files.copy(
+        Paths.get(MODEL_DIR, MII_BASIC_WDT_MODEL_FILE),
+        Paths.get(modelsPath1.toString(), MII_BASIC_WDT_MODEL_FILE),
+        StandardCopyOption.REPLACE_EXISTING));
+
+    // unzip WDT installation file into work dir
+    unzipWDTInstallationFile(errorpathAIPath1.toString());
+
+    // create image with model and wdt installation files
+    createAuxiliaryImage(errorpathAIPath1.toString(),
+        Paths.get(RESOURCE_DIR, "auxiliaryimage", "Dockerfile").toString(), errorPathAuxiliaryImage1);
+
+    // push image to repo for multi node cluster
+    if (!DOMAIN_IMAGES_REPO.isEmpty()) {
+      logger.info("docker push image {0} to registry {1}", errorPathAuxiliaryImage1, DOMAIN_IMAGES_REPO);
+      assertTrue(dockerPush(errorPathAuxiliaryImage1),
+          String.format("docker push failed for image %s", errorPathAuxiliaryImage1));
+    }
+
+    // create domain custom resource using auxiliary images
+    logger.info("Creating domain custom resource with domainUid {0} and auxiliary image {1}",
+        domainUid, errorPathAuxiliaryImage1);
+    Domain domainCR = createDomainResource(domainUid, errorpathDomainNamespace,
+        WEBLOGIC_IMAGE_NAME + ":" + WEBLOGIC_IMAGE_TAG, adminSecretName, OCIR_SECRET_NAME,
+        encryptionSecretName, replicaCount, "cluster-1", auxiliaryImagePath,
+        auxiliaryImageVolumeName, errorPathAuxiliaryImage1);
+
+    // create domain and verify it is failed
+    logger.info("Creating domain {0} with auxiliary image {1} in namespace {2}",
+        domainUid, errorPathAuxiliaryImage1, errorpathDomainNamespace);
+    assertDoesNotThrow(() -> createDomainCustomResource(domainCR), "createDomainCustomResource throws Exception");
+
+    // check the domain event contains the expected error msg
+    String expectedErrorMsg = "Failed to complete processing domain resource domain1 due to: "
+        + "Auxiliary Image: Dir '/errorpath' doesn't exist or is empty. Exiting.";
+    verifyDomainEventContainsExpectedErrorMsg(timestamp, expectedErrorMsg);
```

Comment:
Any error on operator/introspection  logs ?

Reply:
added checking introspector logs and operator logs.